app/services/platform-apps/index.ts

```diff
@@ -73,11 +73,26 @@ interface IAppManifest {
   authorizationUrls: string[];
 }
 
+interface IProductionApp {
+  app_token: string;
+  cdn_url: string;
+  description: string;
+  icon: string;
+  id_hash: string;
+  is_beta: boolean;
+  manifest: IAppManifest;
+  name: string;
+  screenshots: string[];
+  subscription: ISubscription;
+  version: string;
+}
+
 export interface ILoadedApp {
   id: string;
   manifest: IAppManifest;
   unpacked: boolean;
   appPath?: string; // The path on disk to the app if unpacked
+  appUrl?: string; // the cdn url to the unzipped build
   appToken: string;
   devPort?: number; // The port the dev server is running on if unpacked
   poppedOutSlots: EAppPageSlot[];
```

Comment:
I'm thinking now would be a good opportunity to split this interface to get more robust type checking.  It would look like this:

```TypeScript
// Attributes required by all loaded apps
interface IBaseLoadedApp {
  id: string;
  manifest: IAppManifest;
  unpacked: boolean;
  appToken: string;
  poppedOutSlots: EAppPageSlot[];
}

interface IUnpackedLoadedApp extends IBaseLoadedApp {
  unpacked: true;
  appPath: string;
  devPort: number;
}

interface IProductionLoadedApp extends IBaseLoadedApp {
  unpacked: false;
  appUrl: string;
}
```

This pattern is really nice because it makes it impossible to load an unpacked app but provide an appUrl instead of an appPath for example.

Reply:
👍 like this. Will keep in mind to do similar pattern @andycreeth